controllers/fenceagentsremediation_controller_test.go

```diff
@@ -271,6 +357,22 @@ func createVA(vaName, unhealthyNodeName string) *storagev1.VolumeAttachment {
 	return va
 }
 
+// deleteVA delete the specified volume attachment
+func deleteVA(vaName, nodeName string) {
```

Comment:
I think function `deleteVA` could be minimized, since there is no need to create the _ObjectMeta_ and _Spec_ of an already created VA in order to delete it.
In line 234 we can assign the VA to a variable (`va1 := createVA(vaName1, workerNode)`) and then in line 252 simply call `deleteVA(va1)` to run the _ExpectWithOffset_ command.